config/database.yml

```diff
@@ -17,4 +17,7 @@ development:
   database: exercism_development
 
 production:
+  adapter: postgresql
+  encoding: unicode
   url: <%= ENV["DATABASE_URL"] %>
+  database: exercism_production
```

Comment:
These are some hacks I was trying to get the postgres working on heroku but it did not work. Will remove these.